docs/tutorials/notebooks/Unbalanced_OT.ipynb

```diff
@@ -0,0 +1,381 @@
+{
```

Comment:
<p>not very informative IMO without a discussion about this. Say that since the points are ordered per cluster a block diagonal structure would be more natural</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/291/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>